monkey-patch.js

```diff
@@ -88,8 +88,16 @@ function patchAudioContext(NativeAudioContext) {
 function patchOfflineAudioContext(NativeOfflineAudioContext) {
   class OfflineAudioContext extends NativeOfflineAudioContext {
     constructor(...args) {
-      super(...args);
-      console.log(args);
+      // handle initialisation with either an options object or a sequence of parameters
+      // https://developer.mozilla.org/en-US/docs/Web/API/OfflineAudioContext/OfflineAudioContext#parameters
```

Comment:
could you replace the link with this one https://webaudio.github.io/web-audio-api/#dom-offlineaudiocontext-constructor-contextoptions-contextoptions? mdn is very nice, but the spec in more precise